python/ray/dashboard/modules/snapshot/tests/test_actors.py

```diff
@@ -50,16 +50,13 @@ def _kill_actor_using_dashboard_gcs(
     return resp_json
 
 
-@pytest.mark.parametrize("enable_concurrency_group", [False, True])
```

Comment:
This is added by https://github.com/ray-project/ray/pull/51414/. It's no longer needed.